src/migrator.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+
+import Command from 'cli-engine-command'
+import Plugins from './plugins'
+import path from 'path'
+import fs from 'fs-extra'
+
+export default class extends Command {
+  plugins = new Plugins(this)
+
+  async run () {
+    let file = path.join(this.config.dirs.data, '..', 'heroku', 'plugins', 'plugins.json')
+    let pljson
+    try {
+      pljson = fs.readJSONSync(file)
+    } catch (err) {
+      this.debug(err.message)
+      return
+    }
+    for (let p of pljson) {
+      try {
+        // this.log('migrating V5 plugins')
+        if (p.tag === 'symlink') {
```

Comment:
@RasPhilCo could you do a check here to see if the plugin has already be migrated?  (will elaborate more in a later comment)